### DISCUSSION TITLE
How to combine time scoping and space scoping?

### DISCUSSION THREAD

**Discussion by natter1** [ansys/pydpf-core#457] [Q&A] [answered]
I have a transient simulation. Now I want to get the eqv. plastic strain along a path (named selection of nodes) for all time steps. Its a similar problem to [pyansys/pydpf-post/issues/42] (its still not possible with dpf_post?).

In the end I need a plot and also the coordinates of all data points with the corresponding plastic strain value. I have no idea how to combine time and space scoping. And how to get the actual values in the end.

**Comment by PProfizi** [comment, marked as answer]
Hi @natter1, one first step to answering your question is the way to combine time and space scoping. 

Could you please try the following code to get the field container of plastic strain scoped on all time frequencies and on a named_selection 'NAMED_SELECTION':
`plastic_strain_fc = model.results.plastic_strain.on_all_time_freqs.on_named_selection("NAMED_SELECTION").eval()`

You can then plot the last state using:
`plastic_strain_fc[-1].plot()`

As for the coordinates, do you need them updated or not?

**Comment by natter1** [comment]
Hi @PProfizi, sorry for the late reply. Thanks for the solution. My mistake was to seperate both operators and call eval() in between.
